packages/react-components/src/components/MessageStatusIndicator.tsx

```diff
@@ -44,6 +46,8 @@ export interface MessageStatusIndicatorStrings {
 export interface MessageStatusIndicatorProps {
   /** Message status that determines the icon to display. */
   status?: MessageStatus;
+  messageThreadReadCount?: number;
```

Comment:
Minor: some comment for both these new props so the api doesn't complain about this being undocumented.